modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -379,15 +383,19 @@ def concatenate_address(address_line_one, address_line_two, address_line_three)
       def zip
         zip_first_five = @auto_claim&.dig('veteranIdentification', 'mailingAddress', 'zipFirstFive') || ''
         zip_last_four = @auto_claim&.dig('veteranIdentification', 'mailingAddress', 'zipLastFour') || ''
+        international_zip = @pdf_data&.dig(:data, :attributes, :identificationInformation, :mailingAddress, :internationalPostalCode)
         zip = if zip_last_four.present?
                 "#{zip_first_five}-#{zip_last_four}"
+              elsif international_zip.present?
+                international_zip
               else
                 zip_first_five
               end
         mailing_addr = @pdf_data&.dig(:data, :attributes, :identificationInformation, :mailingAddress).present?
```

Comment:
@acovrig Since we have the zip logic in two places it would be nice to abstract it. Maybe another PR after this one is merged, since we are on a deadline.